### DISCUSSION TITLE
quick clarification please

### DISCUSSION THREAD

**Discussion by bobbypfreely** [ArminJo/Arduino-BlueDisplay#10] [Q&A] [answered]
I need a library that works as a bt to android so I can view the serial written from a hall sensor. I have my code your program caught my eye because of all the bells and the whistles. can you tell me if all I need it to do is show me 2 digits read at 100ms at a time. I thought of a speedometer when i read the description to be honest, which if thats possible please explain how to implement. so its clear i have a hal sensor that converts rpm to mph thru  bunch of converting rom inches to minutes to miles and hour, to print in serial the mph of whatever  the sensor is connected to. however i cannot get a good reaing while connected to usb on computr need to bluetooth or led and bt won

**Comment by ArminJo** [comment, marked as answer]
> when i read the description

Which description do you mean?

Printing text is easy. Have a look in the library function [BlueDisplay::printVCCAndTemperaturePeriodically](https://github.com/ArminJo/Arduino-BlueDisplay/blob/536a2fa38369318bc8b4768025b6ceac71acf86e/src/BlueDisplay.hpp#L1525) how to use drawText().

**Comment by bobbypfreely** [comment]
I'm printing the mph converted from a hal effect sensor. I just need it to
display the output of serial port, I think, in the ide it's monitor. But I
got the description from the ide and realized after it was reading the
phones sensors, mine is it's own circuit
